indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -245,3 +264,73 @@ def _get_form_defaults(self):
 class RHSessionBlocks(RHManageSessionBase):
     def _process(self):
         return jsonify_template('events/sessions/management/session_blocks.html', sess=self.session)
+
+
+class RHManageSessionTypes(RHManageSessionsBase):
+    """Dialog to manage the SessionTypes of an event"""
```

Comment:
I'd go for "session types" instead of "SessionTypes" as you do in `RHManageSessionTypeBase` since this is an explanatory docstring.